ui/jobs/jobs_config.js

```diff
@@ -16,8 +17,9 @@ UserConfig.registerOptions('jobs', {
       id: 'LowerOpacityOutOfCombat',
       name: {
         en: 'Lower ui opacity when out of combat',
+        de: 'Veringere die Deckkraft auserhalb des Kampfes',
         fr: 'Diminiuer l\'opacité de l\'UI si hors combat',
-        cn: '非战斗状态时使UI透明化',
+        cn: '仅显示团辅监控',
```

Comment:
Did you mean to edit this chinese text?

Reply:
not really tbh and honnestly dont know where this comes from o.o

Reply:
not really tbh and honnestly dont know where this comes from o.o but its now reverted